src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -1095,7 +1095,6 @@ void QgsIdentifyResultsDialog::addFeature( QgsRasterLayer *layer,
     {
       auto valueLabel { qgis::make_unique<QLabel>( links ) };
       valueLabel->setOpenExternalLinks( true );
-      tblResults->setItem( j, 3, item );
```

Comment:
I think this should be `tblResults->setItem( j, 3, new QTableWidgetItem( it.value() ) )`, just like the `!foundLinks` branch. But then we could just move the `tblResults->setItem( j, 3, new QTableWidgetItem( it.value() ) )` outside of the `if ( foundLinks )` condition entirely...

Reply:
good point. done